contrib/docker/Dockerfile

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+ARG DOCKER_PROXY=""
+FROM ${DOCKER_PROXY}rockylinux:9
+
+ARG GRADLE_VERSION=7.5.1 THRIFT_VERSION=0.17.0 MAVEN_VERSION=3.8.6
+ARG uid
+ARG gid
+
+RUN yum -y groupinstall "Development Tools" && \
+    yum install -y ant wget unzip java-11-openjdk-devel && \
+    cd /tmp && \
+    wget -O gradle.zip https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip && \
+    mkdir /opt/gradle && \
+    unzip -d /opt/gradle gradle.zip && \
+    wget -O thrift.tar.gz https://archive.apache.org/dist/thrift/${THRIFT_VERSION}/thrift-${THRIFT_VERSION}.tar.gz && \
+    tar zxf thrift.tar.gz && \
+    cd thrift-${THRIFT_VERSION} && \
+    export JAVA_HOME=/usr/lib/jvm/java-11-openjdk && \
+    PATH=/opt/gradle/gradle-${GRADLE_VERSION}/bin:${PATH} ./configure --without-kotlin --without-python --without-py3 && \
```

Comment:
You can just do: `--without-libs`, I think. You shouldn't even need ant or Gradle or any of that stuff to build the libthrift.jar java library at all. You only need to build the thrift compiler/generator.

Reply:
nice, I didn't know that.

Reply:
I added `--without-libs` and remove the other `--without*` arguments and the build failed with something having to do with kotlin. Adding `--without-libs` to the current set of arguments did work and allowed me to remove gradle and ant from the Dockerfile.

Reply:
There must be some bug with the kotlin test stuff. If you have the error message, I'm sure we can fix it upstream easily. The kotlin stuff is actively being maintained. It was probably just a simple mistake. You probably only need `--without-kotlin` and `--without-libs`.